server/src/main/java/dev/keva/server/command/impl/kql/manager/KqlManager.java

```diff
@@ -0,0 +1,354 @@
+package dev.keva.server.command.impl.kql.manager;
+
+import net.sf.jsqlparser.JSQLParserException;
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.operators.relational.ExpressionList;
+import net.sf.jsqlparser.parser.CCJSqlParserUtil;
+import net.sf.jsqlparser.schema.Column;
+import net.sf.jsqlparser.schema.Table;
+import net.sf.jsqlparser.statement.Statement;
+import net.sf.jsqlparser.statement.create.table.ColumnDefinition;
+import net.sf.jsqlparser.statement.create.table.CreateTable;
+import net.sf.jsqlparser.statement.delete.Delete;
+import net.sf.jsqlparser.statement.insert.Insert;
+import net.sf.jsqlparser.statement.select.OrderByElement;
+import net.sf.jsqlparser.statement.select.PlainSelect;
+import net.sf.jsqlparser.statement.select.Select;
+import net.sf.jsqlparser.statement.select.SelectItem;
+import net.sf.jsqlparser.statement.update.Update;
+import net.sf.jsqlparser.util.TablesNamesFinder;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class KqlManager {
+    private final Map<String, List<KevaColumnDefinition>> tableMetadata = new ConcurrentHashMap<>();
```

Comment:
Question: Are we going to process queries in single thread manner? If yes then we can change Map implementation to improve performance.
Ex: `HashMap` offer a better performance than `ConcurrentHashMap` in single thread
https://stackoverflow.com/questions/1378310/performance-concurrenthashmap-vs-hashmap

Reply:
@haphananhtuan Thanks for pointing out the issue, I've just changed to use `kevaDatabase`, it will be persisted along with kdb.dump